canvas_modules/common-canvas/src/common-properties/panels/control/control.scss

```diff
@@ -25,6 +28,14 @@
 	}
 }
 
-.properties-editor-form .properties-control-panel .properties-control-panel {
-	padding: 0 0 0 $spacing-03; // Ensure grandchildren have a padding
+.properties-control-panel .properties-control-panel:not(:empty) {
+	// Total 40px spacing between panels
+	padding-bottom: $spacing-06; // (24px + 16px from properties-ctrl-wrapper)
+	&:last-child {
+		padding-bottom: 0;
+	}
+}
+
+.properties-control-panel .properties-control-panel .properties-control-panel {
```

Comment:
What is this styling used for?  A comment might be good here.

Reply:
Removed this styling. It was not used anywhere.